packages/editor-ui/src/Interface.ts

```diff
@@ -856,8 +856,19 @@ export interface INodeMetadata {
 	parametersLastUpdatedAt?: number;
 }
 
-export interface IRootState {
+export interface IExecutionsState {
 	activeExecutions: IExecutionsCurrentSummaryExtended[];
+	executionId: string | null;
+	executingNode: string | null;
+	executionWaitingForWebhook: boolean;
+	saveDataErrorExecution: string;
+	saveDataSuccessExecution: string;
+	saveManualExecutions: boolean;
+	executionTimeout: number;
+	maxExecutionTimeout: number;
```

Comment:
all of these are workflow flags for default values for executing workflows.. not related to directly to previous executions... so these are better suited for settings module and a rename probably...